ILRepack/RepackOptions.cs

```diff
@@ -79,8 +78,10 @@ public List<string> AllowedDuplicateNameSpaces
         private readonly RepackAssemblyResolver globalAssemblyResolver = new RepackAssemblyResolver();
         private List<Regex> excludeInternalizeMatches;
 
-        public void SetSearchDirectories(string[] dirs)
+        public void SetSearchDirectories(IEnumerable<string> dirs)
         {
+            if (TargetPlatformDirectory != null)
+                dirs = new[] { TargetPlatformDirectory }.Concat(dirs);
```

Comment:
Roger that. Will do (if mono, use both the specified folder and the Facades subfolder).
Any advice on how to discover automagically mono fwk folder (like on windows) based on the runtime version ?

Thx !